### ISSUE TITLE
The cursor is misplaced when I enter bad command

### ISSUE THREAD

**Issue by theiThomas** [zeit/hyper#1355]
- **OS version and name**: macOS Sierra
- **Hyper.app version**: 
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: 
- **Relevent information from devtools** _(CMD+SHIFT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: 
- **The issue is reproducible in vanilla Hyper.app**: 

## Issue

When I enter bad command, the cursor is misplaced :
[
![capture d ecran 2017-01-07 a 18 47 19](https://cloud.githubusercontent.com/assets/2513160/21743797/0b59ccfc-d50a-11e6-8c25-f856ba708ef0.png)
](url)

**Comment by timothyis** [comment]
@theiThomas Is this still the case on the most recent release?

**Comment by theiThomas** [comment]
yes, it's still the case. But not in the same use case, now it's when i am in a git repository : 
<img width="1597" alt="capture d ecran 2018-04-23 a 09 33 01" src="https://user-images.githubusercontent.com/2513160/39112634-5e40c9c4-46d9-11e8-9fe1-fd90c7520dcf.png">
